app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,103 @@
+<section class="detail-container">
+
+<h1 class="recipe-name"><%= @recipe.name %></h1>
+<div class="recipe-info">
+  <div class="recipe-info-item1">
+    <div class="desc">
+        <p>Preparation Time: <%= @recipe.preparation %></p>
+        <p>Cooking Time: <%= @recipe.cooking_time %> hours</p>
+        <p>Steps go here...</p>
+    </div>
+    <div class="toggle">
+   <%= button_to "Toggle", toggle_public_recipe_path(@recipe), method: :post, class: "toggle-btn hide-text" %>
+</div>
+
+  
+  </div>
+  <div class="recipe-info-itemBtn">
+    <%= link_to "Generate Shopping List", "#", class: "select-inventory add-ingredient", data: { toggle: "modal", target: "#inventoryModal" } %>
+   <%= link_to 'Add Ingredients', new_recipe_recipe_food_path(@recipe) class: "add-ingredient" %>
+  
+</div>
+
+<table class="recipe-table">
+  <thead>
+    <tr>
+      <th>Food</th>
+      <th>Quantity</th>
+      <th>Value</th>
+      <th>Actions</th>
+    </tr>
+  </thead>
+  <tbody>
+  <% @recipe_foods.each do |recipe_food| %>
```

Comment:
- [ ] Also note that as requested by the previous reviewer, the foods page is missing, kindly add the foods page so as to meet the given requirements

![image](https://github.com/bill7pearl/recipe-rails/assets/81932478/0219c966-74d0-4eb9-904b-836d4b39f964)

Your Design
![image](https://github.com/bill7pearl/recipe-rails/assets/81932478/96aafece-64ab-4913-9c02-0f77c3125044)